compiler/scenario-service/client/src/DA/Daml/LF/PrettyScenario.hs

```diff
@@ -230,15 +230,17 @@ prettyScenarioErrorError (Just err) =  do
                   scenarioError_ContractNotActiveContractRef
         ]
     ScenarioErrorErrorUpdateEmptyContractKeyMaintainers ScenarioError_EmptyContractKeyMaintainers{..} ->
-      pure $
-        "Attempt to create a contract key with an empty set of maintainers in:"
-          $$ nest 2
-          (   "create"
-          <-> prettyMay "<missing template id>" (prettyDefName world) scenarioError_EmptyContractKeyMaintainersTemplateId
-          $$ (   keyword_ "with"
-              $$ nest 2 (prettyMay "<missing argument>" (prettyValue' False 0 world) scenarioError_EmptyContractKeyMaintainersArg)
-             )
-          )
+      pure $ vcat
+        [ "Attempt to create or operate on a contract key with an empty set of maintainers"
```

Comment:
Would it make sense to distinguish between creating a contract and looking up a contract in this error message?

Reply:
I created two exceptions.